src/define_plugin_sets.h

```diff
@@ -281,6 +281,11 @@ To create/register a plugin, you have to :
     #ifndef USES_P033
       #define USES_P033   // Dummy
     #endif
+
+    #ifndef USES_P098
+      #define USES_P098   // Cache Reader
```

Comment:
This should not be included in the minimal_OTA build.
As a matter of fact, I think it only makes sense to include it where the cache controller is included.

Somethinmg like this at the end of this file:

```c++
#ifdef USES_C016
  #ifndef USES_P098
    #define USES_P098
  #endif
#endif
```